docs/config.md

```diff
@@ -32,12 +32,14 @@ for more details.
 
 ### Instrumented libraries and frameworks
 
-| ID | Library | Instrumentation type |
-|-|-|-|
-| `AspNet` | ASP.NET and ASP.NET Core | source |
-| `GraphQL` | [GraphQL](https://www.nuget.org/packages/GraphQL/) | bytecode |
-| `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | source |
-| `SqlClient` | [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient) and [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient) | source |
+| ID | Library | Supported versions | Instrumentation type |
+|-|-|-|-|
+| `AspNet` | ASP.NET MVC Framework | * | source |
```

Comment:
It seems that we need an issue to clarify those `*`. Is it something documented on their original repos?

Reply:
#636